### PR TITLE
test(integration-karma): add integration tests for EventTarget

### PR DESCRIPTION
**Pull request by abdulsattar**
## Details
addEventListener, removeEventListener, dispatchEvent

Co-Authored-By: Sumit Jaiswal <jaiswal.sumit35@gmail.com>

## Does this PR introduce breaking changes?

* ✅ `No, it does not introduce breaking changes.`

## The PR fulfills these requirements:
* Have tests for the proposed changes been added? ✅
* Have you followed [these instructions](../CONTRIBUTING.md#-commit-message-conventions) to clearly describe the issue being fixed or feature enhanced? ✅

## GUS work item
W-8597107

### CODE REVIEW

File: packages/integration-karma/test/shadow-dom/EventTarget-methods/EventTarget.addEventListener.spec.js [hunk 1]
```diff
@@ -1,11 +1,81 @@
...
+                    }.bind(node),
+                });
+            });
+
+            nodes.button.click();
+
+            expect(log).toEqual(targets);
+        });
+
+        it('should throw error when a listener config is passed for shadow root and host', () => {
```
Comment (ekashida) [inline packages/integration-karma/test/shadow-dom/EventTarget-methods/EventTarget.addEventListener.spec.js:71]: This is probably the only test that shouldn't run in native shadow mode.

File: packages/integration-karma/test/shadow-dom/EventTarget-methods/EventTarget.dispatchEvent.spec.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import Container from 'x/container';
+
+function createShadowTree(parentNode) {
+    const elm = createElement('x-container', { is: Container });
+    elm.setAttribute('data-id', 'x-container');
+    parentNode.appendChild(elm);
+    return extractDataIds(elm);
+}
+
+if (!process.env.NATIVE_SHADOW) {
```
Comment (ekashida) [inline packages/integration-karma/test/shadow-dom/EventTarget-methods/EventTarget.dispatchEvent.spec.js:13]: These tests should be fine running in native shadow mode.

File: packages/integration-karma/test/shadow-dom/EventTarget-methods/EventTarget.removeEventListener.spec.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+import Container from 'x/container';
+
+function createShadowTree(parentNode) {
+    const elm = createElement('x-container', { is: Container });
+    elm.setAttribute('data-id', 'x-container');
+    parentNode.appendChild(elm);
+    return extractDataIds(elm);
+}
+
+if (!process.env.NATIVE_SHADOW) {
```
Comment (ekashida) [inline packages/integration-karma/test/shadow-dom/EventTarget-methods/EventTarget.removeEventListener.spec.js:13]: These tests should be fine running in native shadow mode.

**Review by ekashida — CHANGES_REQUESTED** [review]

**Comment by ravijayaramappa** [comment]
@abdulsattar We have disabled integration test suites for forks(to protect sauce labs account information). Please close this PR, push your feature branch to the lwc repo directly and create a new PR.

**Comment by abdulsattar** [comment]
@ravijayaramappa we don't have write access to this repo. @ekashida can you please grant us write access, we'll create a feature branch to push.

**Comment by Sumitjg** [comment]
@ravijayaramappa shall I update lwc/CONTRIBUTING doc not to use fork repo in the steps mentioned for submitting a pull request ?

**Pull request closed without merging**